src/gnmi_clients/gnmi_get.go

```diff
@@ -0,0 +1,142 @@
+/* Copyright 2017 Google Inc.
```

Comment:
Please get original and patch. Apply this comment to all files pulled from public repo.

Reply:
fixed